packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts

```diff
@@ -182,6 +184,8 @@ async function handleSuccessResponse(
   controller: AbortController
 ) {
   const logger: Logger = response.locals.logger
+  const keyVersion: number = Number(response.header('keyVersion'))
```

Comment:
Based on the suggestion above.
```suggestion
  const keyVersion: number = Number(response.header('Odis-Key-Version'))
```